test/fields-relationship/config.ts

```diff
@@ -162,15 +165,27 @@ export default buildConfig({
         name: 'relation-title',
       },
     });
+
+    const relationTitleIDs = [];
+    await mapAsync([...Array(11)], async (item, idx) => {
+      const doc = await payload.create<RelationTwo>({
+        collection: relationWithTitleSlug,
+        data: {
+          name: `relation-title-${idx}`,
+        },
+      });
+      relationTitleIDs.push(doc.id);
+    });
+
     await payload.create<FieldsRelationship>({
       collection: slug,
       data: {
         relationship: relationOneDocId,
         relationshipRestricted: restrictedDocId,
-        relationshipWithTitle: relationWithTitleDocId,
+        relationshipWithTitle: [relationWithTitleDocId],
       },
     });
-    await mapAsync([...Array(11)], async () => {
+    await mapAsync([...Array(5)], async () => {
```

Comment:
We had 11 on there to help with testing pagination. Was there a reason this was dropped to 5?

Reply:
It was a mistake, will revert it.